omega/omega-transport/omega-transport-servicecomb/pom.xml

```diff
@@ -28,6 +28,10 @@
 
   <artifactId>omega-transport-servicecomb</artifactId>
 
+  <properties>
+    <java-chassis.version>1.0.0-m2</java-chassis.version>
```

Comment:
Java chassis version is used in the root pom, we don't need to define it here.

Reply:
Done